packages/docs/src/routes/docs/cheat/qwik-react/index.mdx

```diff
@@ -5,7 +5,7 @@
 #### ⚡️ Qwik
 
 ```tsx
-export const Component = component$(() => {
+export const HelloWorld = component$(() => {
   return (
     <>
```

Comment:
should i remove this fragment bracket without this it will also work fine.

Reply:
sure, but up to you! no difference :)

Reply:
i have made the changes can you review it?